### DISCUSSION TITLE
Default Image for NULL Fields (Photo / Image / Avatar / etc...)

### DISCUSSION THREAD

**Discussion by RealityGamin** [laravel/nova-issues#5144] [Q&A] [answered]
It would be a nice idea to have a default placeholder image for any null image type of field, especially in the global search.

![image](https://user-images.githubusercontent.com/100086757/206359293-15914040-b918-4a19-bb5c-9b4e13466f6b.png)

**Comment by RealityGamin** [comment]
I'm thinking something simple like a gray background with photo icon from [heroicons](https://heroicons.com/)

``<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
  <path stroke-linecap="round" stroke-linejoin="round" d="M2.25 15.75l5.159-5.159a2.25 2.25 0 013.182 0l5.159 5.159m-1.5-1.5l1.409-1.409a2.25 2.25 0 013.182 0l2.909 2.909m-18 3.75h16.5a1.5 1.5 0 001.5-1.5V6a1.5 1.5 0 00-1.5-1.5H3.75A1.5 1.5 0 002.25 6v12a1.5 1.5 0 001.5 1.5zm10.5-11.25h.008v.008h-.008V8.25zm.375 0a.375.375 0 11-.75 0 .375.375 0 01.75 0z" />
</svg>
``

**Comment by crynobone** [comment, marked as answer]
All avatar should be wrapped with `v-if="selectedResource.avatar"`. It seem that the value is actually not `null` in this case.
